xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -364,283 +362,189 @@ - (IBAction)IRRemoteRightArrowPressed:(UIGestureRecognizer*)sender
-              if (ABS(velocityY) > minVelocity && [self shouldFastScroll])
-                [g_xbmcController.inputHandler sendButtonPressed:8];
-
-              m_lastGesturePoint = gesturePoint;
-            }
-            break;
-          }
-          case UIPanGestureRecognizerDirectionDown:
+          if (fabs(m_lastGesturePoint.y - translation.y) >
+              g_xbmcController.inputHandler.inputSettings.siriRemoteVerticalSensitivity)
           {
-            if ((ABS(m_lastGesturePoint.y - gesturePoint.y) > speed) ||
-                ABS(velocityY) > minVelocity)
-            {
-              [g_xbmcController.inputHandler sendButtonPressed:9];
-              if (ABS(velocityY) > minVelocity && [self shouldFastScroll])
-                [g_xbmcController.inputHandler sendButtonPressed:9];
-
-              m_lastGesturePoint = gesturePoint;
-            }
-            break;
+            CLog::Log(LOGDEBUG, "Input: Siri remote pan up (id: 23) %i",
+                      g_xbmcController.inputHandler.inputSettings.siriRemoteVerticalSensitivity);
+            keyId = 23;
           }
-          case UIPanGestureRecognizerDirectionLeft:
+          break;
+        }
+        case UIPanGestureRecognizerDirectionDown:
+        {
+          if (fabs(m_lastGesturePoint.y - translation.y) >
+              g_xbmcController.inputHandler.inputSettings.siriRemoteVerticalSensitivity)
           {
-            // add 80 px to slow left/right swipes, it matched up down better
-            if ((ABS(m_lastGesturePoint.x - gesturePoint.x) > speed + 80) ||
-                ABS(velocityX) > minVelocity)
-            {
-              [g_xbmcController.inputHandler sendButtonPressed:10];
-              if (ABS(velocityX) > minVelocity && [self shouldFastScroll])
-                [g_xbmcController.inputHandler sendButtonPressed:10];
-
-              m_lastGesturePoint = gesturePoint;
-            }
-            break;
+            CLog::Log(LOGDEBUG, "Input: Siri remote pan down (id: 24)");
+            keyId = 24;
           }
-          case UIPanGestureRecognizerDirectionRight:
+          break;
+        }
+        case UIPanGestureRecognizerDirectionLeft:
+        {
+          if (fabs(m_lastGesturePoint.x - translation.x) >
+              g_xbmcController.inputHandler.inputSettings.siriRemoteHorizontalSensitivity)
           {
-            // add 80 px to slow left/right swipes, it matched up down better
-            if ((ABS(m_lastGesturePoint.x - gesturePoint.x) > speed + 80) ||
-                ABS(velocityX) > minVelocity)
-            {
-              [g_xbmcController.inputHandler sendButtonPressed:11];
-              if (ABS(velocityX) > minVelocity && [self shouldFastScroll])
-                [g_xbmcController.inputHandler sendButtonPressed:11];
-
-              m_lastGesturePoint = gesturePoint;
-            }
-            break;
+            CLog::Log(LOGDEBUG, "Input: Siri remote pan left (id: 25) %i",
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)